test/cctest/node_test_fixture.cc

```diff
@@ -0,0 +1,23 @@
+#include <stdlib.h>
+#include "node_test_fixture.h"
+
+char** NodeTestFixture::createargv(const char* prog,
+                                   const char* arg1,
+                                   const char* arg2) {
+  int prog_len = strlen(prog) + 1;
+  int arg1_len = strlen(arg1) + 1;
+  int arg2_len = strlen(arg2) + 1;
+  char **argv = reinterpret_cast<char**>(malloc(3 * sizeof(char*)));
```

Comment:
Style: `char** argv` and you can probably static_cast here.

I'd encapsulate this in a RAII class so you don't have to free manually.  As well, if you have a fixed number of arguments, I'd do something like this:
```c++
struct Argv {
  Argv(const char* a, const char* b, const char* c)
      : a_(a), b_(b), c_(c) {
    vec_[0] = a_.c_str();
    vec_[1] = b_.c_str();
    vec_[2] = c_.c_str();
    vec_[3] = nullptr;
  }

  char* vec_[4];
  std::string a_;
  std::string b_;
  std::string c_;
};